src/Akka.Streams.Kafka.Tests/BugFix240SupervisionStrategy.cs

```diff
@@ -158,7 +158,6 @@ await Source.From(Enumerable.Range(1, 10))
                 .WithBootstrapServers(Fixture.KafkaServer)
                 .WithStopTimeout(TimeSpan.FromSeconds(1))
                 .WithProperty("auto.offset.reset", "earliest")
-                .WithProperty("enable.auto.commit", "false")
```

Comment:
This setting is supposed to be the default, test should work even without this declared